Overview.bs

```diff
@@ -229,6 +255,10 @@ Opt-In Mechanism {#opt-in}
 
 <em>This section is general to both IFT methods.</em>
 
+(XXX Because IFTP is a protocol and IFTB is a format, I suspect most of this section and the related
+technology questions are superfluous.)
```

Comment:
Agreed I think we'll end up with tech(incremental-patch) used with patch subset and format(new-iftb-format-name) for IFTB. Then there's no need for the incremental-auto/incremental-range mechanism and related text.

Reply:
I agree but am leaving this as-is for now.